dfastmi/messages.UK.ini

```diff
@@ -293,46 +293,18 @@ On which river reach is the measure located?
 Discharge location
```

Comment:
Should these changes also be applied to message.NL.ini?

Reply:
Are some of these actually used in the software?

Reply:
Yes, but while experimenting with the GUI layout, I didn't want to each time create and remove entries in both language files. I've now updated the message.NL.ini file as well.

They should all be used in the software ... there is currently no check on that ... You can find the places where these strings are used by searching for `dfastmi.io.log_text` calls in the text (all keys not starting with "gui_") and for `gui_text` text calls (which automatically adds the prefix "gui_" for recognizability)